src/main/scala/no/ndla/imageapi/service/search/IndexService.scala

```diff
@@ -57,7 +57,10 @@ trait IndexService {
     }
 
     def createIndex(): Try[String] = {
-      val indexName = ImageApiProperties.SearchIndex + "_" + getTimestamp
+      createIndexWithName(ImageApiProperties.SearchIndex + "_" + getTimestamp)
```

Comment:
Husker jeg riktig brukes `createIndexWithName` i elasticsearch tester for å generere en ny index med et annet navn enn det som ligger i `ImageApiProperties.SearchIndex`